nucypher/cli/options.py

```diff
@@ -53,6 +51,9 @@
 option_teacher_uri = click.option('--teacher', 'teacher_uri', help="An Ursula URI to start learning from (seednode)", type=click.STRING)
 _option_middleware = click.option('-Z', '--mock-networking', help="Use in-memory transport instead of networking", count=True)
 
+# Avoid circular input
```

Comment:
This is not a common option - So this is just a semi-temporary location for this flag until the full refactor can be performed.